examples/idos-example-dapp/api/EVM.ts

```diff
@@ -19,7 +19,7 @@ const evmGranteeSigner = new ethers.Wallet(
 const idosGrantee = await idOSGrantee.init({
   chainType: "EVM",
   granteeSigner: evmGranteeSigner,
-  encryptionSecret: ENCRYPTION_SECRET_KEY,
+  encryptionPrivateKey: ENCRYPTION_SECRET_KEY,
```

Comment:
Unsure if you wanna do this now or later:
```suggestion
  recipientEncryptionPrivateKey: ENCRYPTION_SECRET_KEY,
```